src/showingpreviously/cinemas/odeon.py

```diff
@@ -0,0 +1,146 @@
+import json
+import re
+
+from selenium import webdriver
+from datetime import datetime, timedelta
+from typing import Tuple, Iterator
+import showingpreviously.requests as requests
+from showingpreviously.model import ChainArchiver, CinemaArchiverException, Chain, Cinema, Screen, Film, Showing
+
+CINEMAS_URL = 'https://vwc.odeon.co.uk/WSVistaWebClient/ocapi/v1/browsing/master-data/sites'
+FILMS_URL = 'https://vwc.odeon.co.uk/WSVistaWebClient/ocapi/v1/browsing/master-data/films'
+SHOWINGS_URL = 'https://vwc.odeon.co.uk/WSVistaWebClient/ocapi/v1/browsing/master-data/showtimes/business-date/{date}'
+
+JWT_TOKEN = None
+DAYS_AHEAD = 2
+CHAIN = Chain('Odeon')
+
+
+def get_jwt_token() -> str:
+    global JWT_TOKEN
+    if JWT_TOKEN is None:
+        driver = webdriver.Chrome()
```

Comment:
This requires the user to have Chrome installed (not always the case - *cough*). I'd recommand we had a check here to see if this worked successfully, and if it didn't then try with Firefox.

You will need to catch `selenium.common.exceptions.WebDriverException` which is thrown for all the relevant issues:
1. Chrome webdriver (`chromedriver`) not installed in PATH
2. Chrome webdriver (`chromedriver`) installed in PATH but Chrome not installed
3. Firefox webdriver (`geckodriver`) not installed in PATH
4. Firefox webdriver (`geckodriver`) installed in PATH but Firefox not installed

(Actually, I haven't tested that last one since I don't want to uninstall Firefox - so please check that the correct exception is thrown)

Reply:
It will now use firefox if chrome throws an exception.